runtime/doc/provider.txt

```diff
@@ -222,6 +222,23 @@ For example this configuration integrates the tmux clipboard: >
           \   'cache_enabled': 1,
           \ }
 
+Commands can also be specified with lists, which is useful for arguments
```

Comment:
could you instead just change the above example to use lists? the string form is a bad idea anyways.

good to keep the docs concise.